xc/device.ml

```diff
@@ -815,25 +814,26 @@ module NetSriovVf = struct
         | 0L -> Some 1L
         | rate -> Some rate)
     in
-    let open Network_interface.Sriov in
-    let net_sriov_vf_config = {
+    let net_sriov_vf_config = Network_interface.Sriov.{
       mac=Some mac; 
       vlan=vlan; 
       rate=rate_Mbps;}
     in
     begin
-      let ret = Net.Sriov.make_vf_config
-        "net_sriov:make_vf_config" ~pci_address:pci ~vf_info:net_sriov_vf_config in
+      let ret = Network_client.Client.Sriov.make_vf_config
+        (Printf.sprintf "%s: Sriov.make_vf_config" dbg)
```

Comment:
Please _only_ pass in the `dbg` string from the task, as-is.

Reply:
Fixed.